CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 
 ## Changes since v5.1.0
 
+- [#503](https://github.com/oauth2-proxy/oauth2-proxy/pull/503) Implements -real-client-ip-header option to select the header from which to obtain a proxied client's IP (@Izzette)
```

Comment:
Nit: Can we make the flag use a double dash as we are requiring this in the project from now on?
```suggestion
- [#503](https://github.com/oauth2-proxy/oauth2-proxy/pull/503) Implements --real-client-ip-header option to select the header from which to obtain a proxied client's IP (@Izzette)
```

Reply:
Oh really?  Yes of course.

Reply:
Done.